selfdrive/ui/qt/body.h

```diff
@@ -5,7 +5,14 @@
 #include <QPushButton>
 
 #include "common/util.h"
+
+#ifdef SUNNYPILOT
+#include "../../sunnypilot/selfdrive/ui/ui.h"
+#define UIState UIStateSP
+#define uiState uiStateSP
```

Comment:
This doesn't seem right. I have a feeling the second line is not needed

Reply:
Fixed in https://github.com/sunnypilot/sunnypilot/pull/500/commits/6056aa49a47422a6b2c2aefe7807d13b1b09796a